test/jdk/java/net/DatagramSocket/SendPortZero.java

```diff
@@ -69,37 +70,37 @@ public void setUp() throws IOException {
 
         // Addresses
         loopbackAddr = InetAddress.getLoopbackAddress();
-        //wildcardAddr = new InetSocketAddress(0).getAddress();
+        wildcardAddr = new InetSocketAddress(0).getAddress();
 
         // Packets
         // loopback w/port 0
         loopbackZeroPkt = new DatagramPacket(buf, 0, buf.length);
         loopbackZeroPkt.setAddress(loopbackAddr);
         loopbackZeroPkt.setPort(0);
 
-        /*
-        //Commented until JDK-8236852 is fixed
-
         // wildcard w/port 0
         wildcardZeroPkt = new DatagramPacket(buf, 0, buf.length);
         wildcardZeroPkt.setAddress(wildcardAddr);
         wildcardZeroPkt.setPort(0);
 
-        //Commented until JDK-8236807 is fixed
-
         // wildcard addr w/valid port
+        // Referenced in bug 8236105, but not addressed.
+        // So, this scenario is not tested
```

Comment:
Is this scenario planned to be addressed? If there is a bug for that (JDK-8236807?) then it could be mentioned in this comment.

Reply:
That bug looks like the same issue. I will add a comment to the code and a note in the bug report about the test.